charts/prometheus-thanos/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.1"
 description: A Helm chart for thanos monitoring components
 name: prometheus-thanos
-version: 2.10.0
+version: 2.11.0
```

Comment:
If we remove cachesize variable we should rais the version to 3.0.0 as its a breaking change for users.